docker/scripts/wait-for-elasticsearch.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+#!/usr/bin/env bash
```

Comment:
I'd pick either one or the other but _not_ both: either `/bin/bash` or `/usr/bin/env bash`.

However, since there seem to be no bash-specific extensions used in this script, I'd recommend simplifying further:
```sh
#!/bin/sh
```
which will always be installed in the path `/bin/sh` and if it isn't, that's a broken Unix 😝

Reply:
Agreed! It makes sense to use /bin/sh. Updated the code.